invisible_cities/daemons/idaemon.py

```diff
@@ -0,0 +1,15 @@
+from importlib import import_module
+import traceback
+
+
+def invoke_daemon(daemon_name):
+    """Takes a daemon name and returns an instance of the daemon"""
```

Comment:
This docstring contradicts the name of the function. Looking at the code, it seems that the docstring is correct, and the function name is misleading. The function should have a name along the lines of `get_daemon`.

Reply:
agreed